debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlValueConverters.java

```diff
@@ -505,6 +505,17 @@ protected boolean matches(String upperCaseTypeName, String upperCaseMatch) {
         return upperCaseMatch.equals(upperCaseTypeName) || upperCaseTypeName.startsWith(upperCaseMatch + "(");
     }
 
+    /**
+     * Determine if the uppercase form of a column's type is geometry collection independent of JDBC driver or server version.
+     *
+     * @param upperCaseTypeName the upper case form of the column's {@link Column#typeName() type name}
+     * @return {@code true} if the type is geometry collection
+     */
+    protected boolean isGeometryCollection(String upperCaseTypeName) {
+        if (upperCaseTypeName == null) return false;
```

Comment:
Can we agree on not using the brace-less `if` form? I know it's used in this class quite a lot, but I'd highly prefer to get a way from this, as it's so easy to produce nasty bugs due to that. In fact I think this should be added to the CS ruleset. We can handle that all separately, I'll just fix this occurence here while merging the PR.

Reply:
@gunnarmorling I am definitely not a proponent of this style - is is just a copy/paste and specialization of `matches` method. I intentionally did not touch the code.

Reply:
Cool. Filed https://issues.jboss.org/browse/DBZ-1039 for getting rid of this pattern across the code base.